lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -266,6 +266,9 @@ protected function transformCampaign($data) {
         $output['uri'] = $data->uri;
       }
 
+      if (dosomething_helpers_get_variable('node', $data->id, 'magic_link_copy')) {
```

Comment:
@katiecrane the value in the API is `FALSE`? 
the api response should def be `NULL` but maybe I'm not understanding what what is the 'it' in this case?